### PR TITLE
Azure - Slightly more verbose functions docs

### PR DESCRIPTION
**Pull request by stefangordon**

### CODE REVIEW

File: docs/source/azure/advanced/azurefunctions.rst [hunk 1]
```diff
@@ -34,7 +44,26 @@ provisioned. Application Insights has six available locations and thus can not a
...
+.. code-block:: yaml
+
+    policies:
+      - name: stopped-vm
+        mode:
+            type: azure-periodic
+            provision-options:
+              servicePlanName: functionshost
+         resource: azure.vm
+         filters:
```
Comment (aluong) [inline docs/source/azure/advanced/azurefunctions.rst]: Maybe we should add an action here, until we give clear instructions on how to see the output.

File: docs/source/azure/advanced/azurefunctions.rst [hunk 2]
```diff
@@ -34,7 +44,26 @@ provisioned. Application Insights has six available locations and thus can not a
...
 
-An example on how to set the provision-options when running in azure functions mode:
+An example on how to set the servicePlanName and accept defaults for the other values:
+
+.. code-block:: yaml
+
+    policies:
+      - name: stopped-vm
+        mode:
+            type: azure-periodic
```
Comment (thisisshi) [inline docs/source/azure/advanced/azurefunctions.rst]: I'm not super familiar with azure functions so let me know if this isn't applicable, but user-definable schedules are available in the original `periodic` mode, is there a way for the user to define the cron schedule or rate of execution wrt azure periodic functions?

example
```yaml
policies:
  - name: periodic-ec2
    resource: ec2
    mode:
       type: periodic
       schedule: rate(5 minutes)
...
```

Comment (stefangordon) [reply docs/source/azure/advanced/azurefunctions.rst]: Yeah, it is just missing in this example :)   Let me update it with the field documented!  Tx!

File: docs/source/azure/advanced/azurefunctions.rst (hunk 1 above)
Comment (stefangordon) [reply docs/source/azure/advanced/azurefunctions.rst]: Yeah just didn't put any thought into it yet as I haven't ever run an action even, hah.  The docs were mostly for you so far :)   Once it is all working with final naming and stuff we can write a bunch.

**Pull request merged**